src/arkit.cs

```diff
@@ -1045,6 +1158,128 @@ interface ARDirectionalLightEstimate {
 	}
+
+	[iOS (12,0)]
+	[NoWatch, NoTV, NoMac]
+	[BaseType (typeof(ARConfiguration))]
+	interface ARImageTrackingConfiguration {
+		[Export ("autoFocusEnabled")]
+		bool AutoFocusEnabled { [Bind ("isAutoFocusEnabled")] get; set; }
+
+		[Export ("trackingImages", ArgumentSemantic.Copy)]
+		NSSet<ARReferenceImage> TrackingImages { get; set; }
+
+		[Export ("maximumNumberOfTrackedImages")]
+		nint MaximumNumberOfTrackedImages { get; set; }
+	}
+
+	[iOS (12,0)]
+	[NoWatch, NoTV, NoMac]
+	[BaseType (typeof(ARConfiguration))]
+	interface ARObjectScanningConfiguration {
+		[Export ("autoFocusEnabled")]
+		bool AutoFocusEnabled { [Bind ("isAutoFocusEnabled")] get; set; }
+
+		[Export ("planeDetection", ArgumentSemantic.Assign)]
+		ARPlaneDetection PlaneDetection { get; set; }
+	}
+
+	[iOS (12,0)]
+	[NoWatch, NoTV, NoMac]
+	[BaseType (typeof(ARAnchor))]
+	interface AREnvironmentProbeAnchor {
+		[Export ("initWithTransform:extent:")]
+		[MarshalDirective (NativePrefix = "xamarin_simd__", Library = "__Internal")]
+		IntPtr Constructor (Matrix4 transform, Vector3 extent);
+
+		[Export ("initWithName:transform:extent:")]
+		[MarshalDirective (NativePrefix = "xamarin_simd__", Library = "__Internal")]
+		IntPtr Constructor (string name, Matrix4 transform, Vector3 extent);
+
+		[NullAllowed, Export ("environmentTexture", ArgumentSemantic.Strong)]
+		IMTLTexture EnvironmentTexture { get; }
+
+		[Export ("extent")]
+		Vector3 Extent {
+			[MarshalDirective (NativePrefix = "xamarin_simd__", Library = "__Internal")]
+			get;
+		}
+	}
+
+	[iOS (12,0)]
+	[NoWatch, NoTV, NoMac]
+	[BaseType (typeof(NSObject))]
+	[DisableDefaultCtor]
+	interface ARReferenceObject : NSCopying, NSSecureCoding {
+		[Export ("initWithArchiveURL:error:")]
+		IntPtr Constructor (NSUrl archiveUrl, [NullAllowed] out NSError error);
+
+		[NullAllowed, Export ("name")]
+		string Name { get; set; }
+
+		[Export ("center")]
+		Vector3 Center {
+			[MarshalDirective (NativePrefix = "xamarin_simd__", Library = "__Internal")]
+			get;
+		}
+
+		[Export ("extent")]
+		Vector3 Extent {
+			[MarshalDirective (NativePrefix = "xamarin_simd__", Library = "__Internal")]
+			get;
+		}
+
+		[Export ("scale")]
+		Vector3 Scale {
+			[MarshalDirective (NativePrefix = "xamarin_simd__", Library = "__Internal")]
+			get;
+		}
+
+		[Export ("rawFeaturePoints", ArgumentSemantic.Strong)]
+		ARPointCloud RawFeaturePoints { get; }
+
+		[Static]
+		[Export ("referenceObjectsInGroupNamed:bundle:")]
+		[return: NullAllowed]
+		NSSet<ARReferenceObject> GetReferenceObjectsInGroup (string name, [NullAllowed] NSBundle bundle);
+
+		[Export ("exportObjectToURL:previewImage:error:")]
+		bool ExportObject (NSUrl url, [NullAllowed] UIImage previewImage, [NullAllowed] out NSError error);
+
+		[Export ("referenceObjectByApplyingTransform:")]
+		[MarshalDirective (NativePrefix = "xamarin_simd__", Library = "__Internal")]
+		ARReferenceObject GetReferenceObject (Matrix4 transform);
+	}
+
+	[iOS (12,0)]
+	[BaseType (typeof(ARAnchor))]
```

Comment:
the base type has it's default actor marked as not available
unless mentioned explicitly in the headers this should not expose it